lib/OpenQA/WebAPI.pm

```diff
@@ -123,6 +123,7 @@ sub startup {
         before_dispatch => sub {
             my $c = shift;
             #$c->app->log->debug(sprintf "this connection is %ssecure", $c->req->is_secure?'':'NOT ');
+            $c->app->sessions->default_expiration(14 * 24 * 60 * 60);
```

Comment:
Does this has to be in `begore_dispatch` hook? Isn't it sufficient to put it somewhere in the `startup` routime?